### ISSUE TITLE
example_single_task_cnn.py

### ISSUE THREAD

**Issue by zshu000000** [ml-jku/DeepRC#9]
I tried to run the "example_single_task_cnn.py" with the data provided in the "example_dataset" folder and got a ROC_AUC of 0.5, bacc of 0.5, and f1_score of 0.0. Obviously, this is not reasonable, but I don't know what's wrong. Thanks.

**Comment by zshu000000** [comment]
Is there an error in the getitem() method?

**Comment by widmi** [comment]
Hi! Are the scores you reported for the training, validation, or test set?

**Comment by widmi** [comment]
The data in example_single_task_cnn.py is purely random, so there is no signal to learn and the model shouldn't generalize. It should only overfit on the training data but not generalize to the validation or test set.

**Comment by zshu000000** [comment]
Hi, thanks for your reply.  These scores are the scores for the test set. However, when we replace it with our own data, we get the same score.

**Comment by zshu000000** [comment]
We train and test on our own data and get the same scores.

**Comment by widmi** [comment]
So the training set scores are also random? Or just the test set scores?

**Comment by widmi** [comment]
If you get very good training scores but random test scores, the model is overfitting. That's not a bad thing for the vanilla model, it just means that you need to use a smaller network and/or more regularization. In case a smaller network and/or more regularization doesn't help, you'd need more training samples.

**Comment by zshu000000** [comment]
Thanks for your reply
